cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -22,6 +22,11 @@ ARTIFACT_DOWNLOAD_URL="${INSTALLER_DOWNLOAD_URL:-{{ base_url }}}"
 PRINT_VERBOSE=${INSTALLER_PRINT_VERBOSE:-0}
 PRINT_QUIET=${INSTALLER_PRINT_QUIET:-0}
 NO_MODIFY_PATH=${INSTALLER_NO_MODIFY_PATH:-0}
+read -r RECEIPT <<EORECEIPT
+{{ receipt | tojson }}
+EORECEIPT
+# Are we happy with this same path on Linux and Mac?
+RECEIPT_HOME="${HOME}/.config/{{ app_name }}"
```

Comment:
this should probably be scoped to each platform in The Grand Switch, since in principle each platform can have its own binaries (and presumably good to know `.exe` vs not?).

Reply:
When you say "platform", do you mean architecture? EXE shouldn't be relevant here in the shell installer.

Reply:
hmm you're right that's true right now, but it feels like a tenuous truce.

i mean per-"target-triple". The Grand Switch on the detected target-triple already encodes a list of binaries that is Conceptually free to vary per-platform.

Reply:
Now that I realize you're talking about binaries - yeah, this is a bit of a problem should those ever vary per-platform.

I've pushed a commit that puts a placeholder bin array into the template, then updates it with what's actually calculated later in the file. This feels... messy though? Hm.